### DISCUSSION TITLE
Wiki sobre android

### DISCUSSION THREAD

**Discussion by Poseidon2579** [community/community#64535] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Es personal

**Comment by TostadoraDev** [comment]
Necesitas ser pro para hacer una wiki

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
